doc/modules/preprocessing_targets.rst

```diff
@@ -7,6 +6,71 @@
 Transforming the prediction target (``y``)
 ==========================================
 
+Transforming target in regression
+---------------------------------
+
+:class:`TransformedTargetRegressor` is a meta-estimator transforming the target
```

Comment:
yeap... how about "class[...] wraps a regressor and applies a transformation to the target before fitting, finally transforming the regressor's predictions back to the original space" or something